coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/ProofAggregationCoordinatorServiceTest.kt

```diff
@@ -74,7 +78,20 @@ class ProofAggregationCoordinatorServiceTest {
     val mockAggregationCalculator = mock<AggregationCalculator>()
     val mockAggregationsRepository = mock<AggregationsRepository>()
     val mockProofAggregationClient = mock<ProofAggregationProverClientV2>()
-    val aggregationL2StateProvider = mock<AggregationL2StateProvider>()
+    val mockAggregationL2StateProvider = mock<AggregationL2StateProvider>()
+
+    // histogram metrics mocks
+    lateinit var mockAggregationSizeInBlocksHistogram: Histogram
+    lateinit var mockAggregationSizeInBatchesHistogram: Histogram
+    lateinit var mockAggregationSizeInBlobsHistogram: Histogram
+    val mockMetricsFacade: MetricsFacade = mock() {
```

Comment:
Isn't it easier to create a real instance instead? We have the micrometer dependency in the test fixtures anyway